src/sentry/buffer/redis.py

```diff
@@ -289,7 +289,7 @@ def push_to_hash(
 
     def get_hash(
         self, model: type[models.Model], field: dict[str, models.Model | str | int]
-    ) -> dict[str, str]:
+    ) -> dict[str, int]:
```

Comment:
`[{'2:3': '4'}]` - this is how Redis returns it to us. I could turn the value into an int before returning if that helps? For now I'm gonna leave this as it is though.